### ISSUE TITLE
fine-tunning

### ISSUE THREAD

**Issue by Alan-Py** [lucidrains/med-seg-diff-pytorch#16]
Can someone share your pt file,I want to do fine tunning. My loss is too big.
